layouts/analytics/list.html

```
@@ -1,21 +1,4 @@
-  </div>
-  {{ .Content }}
-  <div id="neo-analytics-dashboard"></div>
-  <div id="neo-time-chart-section"></div>
-  <div id="neo-retention-section"></div>
-  <div id="neo-fav-summary" class="neo-fav-summary"></div>
-</div>
-</main>
-{{ partial "neo-footer.html" . }}
-</body>
-</html>
+{{/*
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)